scripts/resources/debian/source/include-binaries

```diff
@@ -1 +1 @@
-debian/tachidesk.png
+debian/faviconlogo.png
```

Comment:
bad naming?

Reply:
It's fine. It will be get renamed in `debian/install` to `tachidesk.png`. I think it's less confusing if I keep original naming as much as possible across different files

Reply:
This is the file that gets copied from `server/src/main/resources/icon/faviconlogo.png` to debian folder